lib/scala/filewatcher/src/main/scala/org/enso/filewatcher/WatcherAdapter.scala

```diff
@@ -12,99 +12,68 @@ import java.nio.file.Path
   */
 final class WatcherAdapter(
   root: Path,
-  eventCallback: WatcherAdapter.WatcherEvent => Unit,
-  errorCallback: WatcherAdapter.WatcherError => Unit
-) extends DirectoryChangeListener {
-
-  import WatcherAdapter._
```

Comment:
The test shouldn't even see this class and definitely not ` io.methvin.watcher` classes on the classpath!